modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -636,6 +636,78 @@ definition {
 		}
 	}
 
+	@description = "LPS-178410 - Verify if actions works with oldValue function when trigger is On After Update (Notification) in Name field"
+	@priority = 4
+	test CanCreateActionNotificationUsingOldValueWithOnAfterUpdateTriggerOnAccountObject {
+		task ("Given: A notification template is created") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test Test",
+				name = "Email Template",
+				notificationType = "Email",
+				subject = "Subject Test");
+
+			PortletEntry.save();
+		}
+
+		task ("And given: A Notification action with the trigger (On After Update) is created and the oldValue function is used in the condition tab") {
```

Comment:
Let's make it clear that the oldValue function will be used for the Account Name